floodrelief/settings.py

```diff
@@ -71,8 +72,24 @@ def get_list(text):
     'rest_framework',
     'rest_framework.authtoken',
     'rest_auth',
+    'ddtrace.contrib.django',
 ]
 
+DATADOG_TRACE = {
+    'DEFAULT_SERVICE': 'django',
+    'TAGS': {'env': 'production'},
+    'AGENT_HOSTNAME': env('DATADOG_HOST'),
+    'AGENT_PORT': '8126',
+    'ENABLED': True,
```

Comment:
Can we disable the tracing for dev based on an env?

Reply:
sure, i believe this is shelved for now. so it wont reflect in development now